core/templates/domain/exploration/SubtitledHtmlObjectFactory.ts

```diff
@@ -17,18 +17,18 @@
  * domain objects.
  */
 
-export interface ISubtitledHtmlBackendDict {
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+export interface SubtitledHtmlBackendDict {
   'content_id': string;
   'html': string;
 }
 
-import { Injectable } from '@angular/core';
-import { downgradeInjectable } from '@angular/upgrade/static';
-
 export class SubtitledHtml {
   _html: string;
-  _contentId: string;
-  constructor(html: string, contentId: string) {
+  _contentId: string | null;
```

Comment:
Woah. Why null? Don't do that, ensure that the constructor includes a contentId (unless there's some reason I'm missing for this not being possible?).